### ISSUE TITLE
Add Blauth grading thumb hypoplasia

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10461]
**Preferred term label:**
Thumb hypoplasia grade 3A

**Synonyms**
Hypoplasia of the thumb grade IIIA

**Definition (free text, please give PubMed ID)**
Musculotendinous and osseous deficiencies with the CMC joint intact. Absence of active motion at the MCP or IP joint
PMID: 24362256

**Parent term (use hpo.jax.org/app)**
Short thumb HP:0009778

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
See the disease associations from Short thumb HP:0009778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000940
- label: Thumb hypoplasia grade 3A
- parents: [Short thumb (HP:0009778)](https://hpo.jax.org/app/browse/term/HP:0009778)
- pmid: PMID:24362256
- def: A type of developmental undergrowth of the thumb, whereby there are musculotendinous and osseous deficiencies with the carpometacarpal (CMC) joint intact. Absence of active motion at the metacarpophalangeal (MCP) or interphalangeal (IP) joint.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10461
- synonyms: Hypoplasia of the thumb grade IIIA(NONE)
